cmd/fleetctl/convert.go

```diff
@@ -19,15 +19,60 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+func convertPlatform(platformsIn string) (*string, error) {
```

Comment:
You should return a `string`, not a string pointer here (does a `nil` string have a different meaning? It doesn't look like it looking at where you call `convertPlatform`).